backend/core/management/commands/reset_db.py

```
@@ -0,0 +1,35 @@
+import os
+import django
```

Ruff

`django` imported but unused

Remove unused import: `django`

(F401)